openapi/openapi.yaml

```diff
@@ -2202,9 +2203,12 @@ components:
         github_installation_id:
           type: integer
           description: GitHub App Installation ID.
-        origin:
```

Comment:
Can we keep origin as discussed?

Reply:
We can but we won't use it :)

Reply:
We will (we already use it) - in syncs to keep track of "origin" of software stacks in a queryable form.

Reply:
right, let me change it :)